app/domain/authentication/authn_jwt/signing_key/public_signing_keys.rb

```diff
@@ -11,11 +11,11 @@ class PublicSigningKeys
         attr_required(:type, :value)
 
         validates(*required_attributes, presence: true)
-        validates(:type, inclusion: { in: VALID_TYPES, message: "'%{value}' is not a valid public-keys type" })
+        validates(:type, inclusion: { in: VALID_TYPES, message:  "'%{value}' is not a valid public-keys type" })
```

Comment:
can you pls add "valid value is jwks"
so the user will have the ability to quickly resolve the issue?

Reply:
Done. See now